dh/sidh/internal/p434/core.go

```diff
@@ -252,7 +252,7 @@ func DeriveSecretA(ss, prv []byte, pub3Pt *[3]Fp2) {
 	// Traverse isogeny tree
 	traverseTreeSharedKeyA(&cparam, &xR)
 
-	// Calculate j-invariant on isogeneus curve
+	// Calculate j-invariant on isogenous curve
```

Comment:
I'm not remotely familiar w/ this term, but Google suggests that it's a typo...

Reply:
`isogenous` is the right spelling